examples/with-facebook-pixel/README.md

```diff
@@ -0,0 +1,23 @@
+## Example app using Facebook Pixel
+
+This example shows how to use Next.js along with Facebook Pixel. A custom `_app.js` is used to track route changes and send pageviews to facebook pixel. This example uses [react-facebook-pixel](https://www.npmjs.com/package/react-facebook-pixel).
+
+You will need to set your `NEXT_PUBLIC_FACEBOOK_PIXEL_ID` environment variable for it to send events to your facebook app. When not specified, tracking is disabled.
```

Comment:
@ashconnell Can you also mention here that you need to create a `.env.local` or `.env` file first. E.g: https://github.com/vercel/next.js/tree/canary/examples/cms-datocms#step-5-set-up-environment-variables

Reply:
Probably better to move this to be under the `create-next-app` step

Reply:
Thanks @lfades this is now resolved